### ISSUE TITLE
run testData branch MONAIAuto3DSegTest  error

### ISSUE THREAD

**Issue by zist8888** [lassoan/SlicerMONAIAuto3DSeg#79]
Dependencies are set up successfully.
Processing started
Writing input file to /tmp/Slicer-ubuntu/__SlicerTemp__2024-09-26_16+31+46.165/input-volume.nrrd
Creating segmentations with MONAIAuto3DSeg AI...
Auto3DSeg command: ['/data/soft/Slicer-5.7.0-2024-03-25-linux-amd64/bin/../bin/PythonSlicer', '/data/soft/Slicer-5.7.0-2024-03-25-linux-amd64/sl                         icer.org/Extensions-32785/MONAIAuto3DSeg/lib/Slicer-5.7/qt-scripted-modules/Scripts/auto3dseg_segresnet_inference.py', '--model-file', '/home/ub                         untu/.MONAIAuto3DSeg/models/abdominal-organs-3mm/model.pt', '--image-file', '/tmp/Slicer-ubuntu/__SlicerTemp__2024-09-26_16+31+46.165/input-volu                         me.nrrd', '--result-file', '/data/soft/Slicer-5.7.0-2024-03-25-linux-amd64/output-segmentation.nii.gz']
`apex.normalization.InstanceNorm3dNVFuser` is not installed properly, use nn.InstanceNorm3d instead.
Model epoch 294 metric 0.9070999026298523
Using crop_foreground
Using resample with  resample_resolution [3.0, 3.0, 3.0]
Running Inference ...

  0%|          | 0/6 [00:00<?, ?it/s]Applied workaround for CuDNN issue, install nvrtc.so (Triggered internally at ../aten/src/ATen/native/cudnn                         /Conv_v8.cpp:84.)

 17%|█▋        | 1/6 [00:00<00:01,  3.50it100%|██████████| 6/6 [00:0100%|██████████| 6/6 [00:0Logits torch.Size([1, 25, 160, 133, 134])
Converting logits into predictions
preds torch.Size([1, 1, 160, 133, 134])
preds inverted torch.Size([512, 512, 321])
Computation time log:
  Loading volumes: 1.35 seconds
  Preprocessing: 2.60 seconds
  Inference: 0.53 seconds
  Logits: 0.33 seconds
  Preds: 0.15 seconds
  Convert to array: 0.32 seconds
  Save: 4.06 seconds
ALL DONE, result saved in /data/soft/Slicer-5.7.0-2024-03-25-linux-amd64/output-segmentation.nii.gz
Processing failed with return code 1002
Cleaning up temporary folder.
Processing failed after 16.58 seconds.
double free or corruption (!prev)
error: [/data/soft/Slicer-5.7.0-2024-03-25-linux-amd64/bin/SlicerApp-real] exit abnormally - Report the problem.

run Test without slicer3D UI，I get error，save file cannt open ，why？tks

**Comment by lassoan** [comment]
Phrase user the latest Slicer Stable Release or latest Slicer Preview Release and key is know if you run into any issues.

**Comment by lassoan** [comment]
The problem is that the result file format is incorrect (you get .nii.gz, while .nrrd file format should be used instead).

**Comment by zist8888** [comment]
you are right,i have modified at the afternoon,thanks for replying!
